include/bbp/sonata/report_reader.h

```diff
@@ -127,8 +127,19 @@ class SONATA_API ReportReader
          * Return true if the data is sorted.
          */
         bool getSorted() const;
+
+        /**
+         * Return all the node ids.
+         */
         std::vector<NodeID> getNodeIds() const;
 
+        /**
+         * Return selected ids.
+         */
+        typename DataFrame<KeyType>::DataType getIds(
```

Comment:
should there be a python counterpart to this?

Reply:
Yes, but i will avoid the std::function xD

Reply:
Yup, makes sense :)

Reply:
Any luck with this?